src/proto/streams/recv.rs

```diff
@@ -246,6 +250,19 @@ where
         Ok(())
     }
 
+    pub fn set_target_connection_window(&mut self, target: WindowSize, task: &mut Option<Task>) {
+        let current = self.flow.available() + self.in_flight_data;
+        if target > current {
+            self.flow.assign_capacity(target - current);
+
+            if self.flow.unclaimed_capacity().is_some() {
+                if let Some(task) = task.take() {
```

Comment:
For my benefit (as I am losing context of the code already), could we add a comment here indicating that `unclaimed_capacity` being `Some` means that we have passed the window threshold to send a `WINDOW_UPDATE` (we don't always want to send a `WINDOW_UPDATE` for small deltas). And the `task` here is the task that manages the HTTP/2.0 connection, which is responsible for sending out the window updates.